src/Domain/CoordinateMaps/FocallyLiftedMapHelpers.cpp

```diff
@@ -0,0 +1,215 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/FocallyLiftedMapHelpers.hpp"
+
+#include <boost/none.hpp>
+#include <cmath>
+#include <gsl/gsl_poly.h>
+#include <limits>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/RootFinding/QuadraticEquation.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace domain::CoordinateMaps::FocallyLiftedMapHelpers {
+
+template <typename T>
+tt::remove_cvref_wrap_t<T> scale_factor(
+    const std::array<T, 3>& src_point, const std::array<double, 3>& proj_center,
+    const std::array<double, 3>& sphere_center, double radius,
+    const bool src_is_between_proj_and_target) noexcept {
+  using return_type = tt::remove_cvref_wrap_t<T>;
+  // quadratic equation is
+  // a x^2 + b x + c = 0
```

Comment:
Here you are not solving for scale_factor - 1 ... why not? Aren't you concerned about the same roundoff issue as in try_scale_factor? 

If you do want to keep both versions, please add a derivation of both to the help text, since they are so similar it's easy for a reader to confuse one for the other